src/test/java/fi/aalto/cs/apluscourses/intellij/notifications/NetworkErrorNotificationTest.java

```diff
@@ -0,0 +1,23 @@
+package fi.aalto.cs.apluscourses.intellij.notifications;
+
+import static org.hamcrest.Matchers.containsString;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class NetworkErrorNotificationTest {
+
+  @Test
+  public void testNetworkErrorNotification() {
+    Exception exception = new Exception("hello there");
```

Comment:
seems, like this is testing two things: title concatenation (in a way) and also the method that extracts the message from the `Exception`. if this is the intention, then a small improvement could be in externalizing the `"hello there"` into a `String`, passing it to the newly created exception, and then checking that same `String` availability from the assert. this way, if the behaviour of `exception.getMessage()` changes (half of the Java code in the world would break), but the test case will indicate as well :stuck_out_tongue_winking_eye: